src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -73,6 +75,9 @@ export class MdProgressSpinner implements OnDestroy {
   private _value: number;
   private _color: string = 'primary';
 
+  /** Stroke width of the progress spinner. By default uses 10px as stroke width. */
```

Comment:
I don't think that this should have the `By default uses 10px as stroke width.` since the default may change and I doubt that we'd remember to update the comment.